proposals/geotagging/webvmt/index.html

```diff
@@ -3347,7 +3350,10 @@ <h2>WebVMT Cue Timings Parsing</h2>
               If |position| is past the end of |input|, return an error and abort these steps.
             </li>
             <li>
-              If the character indicated by |position| is not an <a data-cite='HTML51/infrastructure.html#ascii-digit'>ASCII digit</a>, then return an error and abort these steps.
+              If the character at |position| is a U+002D HYPHEN-MINUS character (-), let |negative| be true and move |position| forwards one character. Otherwise, let |negative| be false.
```

Comment:
Conforming to latest [ Character naming template](https://www.w3.org/guide/manual-of-style/#char_ref_template) in the [W3C Manual of Style](https://www.w3.org/guide/manual-of-style/)
```suggestion
              If the character at |position| is a <span class="codepoint" translate="no"><bdi lang="en">&#002D;</bdi><span class="uname">U+002D HYPHEN-MINUS</span></span>, let |negative| be true and move |position| forwards one character. Otherwise, let |negative| be false.
```

Reply:
@TallTed Thanks for your feedback. Apologies. I wasn't expecting this minor change to be contentious and your comment crossed with my merge confirmation.

[W3C Manual of Style, section 4.2 Unicode](https://www.w3.org/guide/manual-of-style/#Unicode) states:
> Use the Unicode character name to describe specific code points. Use of the [character naming template](https://www.w3.org/guide/manual-of-style/#char_ref_template) is recommended.

Yes, I agree in principle. However, the [WebVMT Editor's Draft](https://w3c.github.io/sdw/proposals/geotagging/webvmt/) contains many such references, so I propose to defer your suggested change from this PR, and make all 100+ changes together in a separate PR for consistency if that's an acceptable alternative.

Reply:
Updated Unicode code points in #1454 as promised